packages/cozy-harvest-lib/src/konnector-policies.js

```diff
@@ -26,8 +32,5 @@ logger.info('Available konnector policies', policies)
  * @param {KonnectorManifest} konnector
  * @returns {KonnectorPolicy}
  */
-export const findKonnectorPolicy = konnector => {
-  const policy = policies.find(policy => policy.match(konnector))
-  logger.info(`Using ${policy.name} konnector policy for ${konnector.slug}`)
-  return policy
-}
+export const findKonnectorPolicy = konnector =>
```

Comment:
Why did you remove the log?

Reply:
This generates too many lines of logs in a simple connector execution. I now moved this removal in its own commit with explanations

Reply:
Maybe it's a hint that there are too many call / re-renders ? Not sure that hiding it is the right solution. 

Is it intentional to have so many calls to this method?

Reply:
Since this method can be called with different konnectors  in the same session, it looks like it's mandatory to call it everytime we need a policy.

At first I thought we could define a an association object on load and use it for the next calls, but since the user can install, update konnectors in the interval, we cannot be sure that the same policy will be used for the same konnector everytime.